src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -46,13 +46,50 @@ class ActivityFeed extends React.Component<Props, State> {
 
     feedContainer: null | HTMLElement;
 
+    componentDidMount() {
+        this.resetFeedScroll();
+    }
+
+    componentDidUpdate(prevProps: Props, prevState: State) {
+        const {
+            currentUser: previousUser,
+            feedItems: prevFeedItems,
+        } = prevProps;
+        const { currentUser, feedItems: currFeedItems } = this.props;
+        const { isInputOpen: prevIsInputOpen } = prevState;
+        const { isInputOpen: currIsInputOpen } = this.state;
+
+        const hasMoreItems =
+            prevFeedItems &&
+            currFeedItems &&
+            prevFeedItems.length < currFeedItems.length;
+        const hasNewItems = !prevFeedItems && currFeedItems;
+        const hasNewUser = currentUser !== previousUser;
+        const hasInputOpened = currIsInputOpen !== prevIsInputOpen;
```

Comment:
Can the input open such that the focus handler on the ApprovalCommentForm won't be called? Otherwise we may be able to remove that case.

Reply:
I don't think so. It looks like the only case of the input being set to open is on L96 - this.setState({ isInputOpen: true })